coops-ui/src/App.vue

```diff
@@ -358,7 +354,5 @@ export default {
 </script>
 
 <style lang="scss">
-@import "@/assets/styles/base.scss";
-@import "@/assets/styles/layout.scss";
-@import "@/assets/styles/overrides.scss";
+// @import '@/assets/styles/theme.scss';
```

Comment:
Love it

Reply:
FYI, it might seems weird to import the SCSS in main.ts instead (in fact I prefer to keep presentation stuff together instead of mixing it with code) but it's the way for style order of precedence to work as intended.